### PR TITLE
add ng-bootstrap modal

### PR DESCRIPTION
**Pull request by aperepeliak**

### CODE REVIEW

File: src/app/core/services/modal/modal.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  constructor(private modalService: NgbModal) {}
+
+  confirm(itemName: string): Observable<boolean> {
+    const modalRef = this.modalService.open(ConfirmDeleteModalComponent);
+
+    modalRef.componentInstance.name = itemName;
+
+    const result = modalRef.result;
+
+    return Observable.create((observer: Observer<boolean>) => {
```
Comment (kamiua) [inline src/app/core/services/modal/modal.service.ts]: Why not use `fromPromise`?

File: src/app/core/services/modal/modal.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import { ConfirmDeleteModalComponent } from '../../components';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ModalService {
+  constructor(private modalService: NgbModal) {}
+
+  confirm(itemName: string): Observable<boolean> {
```
Comment (kamiua) [inline src/app/core/services/modal/modal.service.ts]: Consider renaming method to reflect that it is delete confirmation.

File: src/app/core/services/modal/modal.service.ts (hunk 1 above)
Comment (aperepeliak) [reply src/app/core/services/modal/modal.service.ts]: haven't heard :blush:

File: src/app/core/services/modal/modal.service.ts (hunk 2 above)
Comment (aperepeliak) [reply src/app/core/services/modal/modal.service.ts]: Sure!

**Pull request merged**